app/services/encryption/scrypt_digest_maker.rb

```diff
@@ -0,0 +1,5 @@
+module Encryption
```

Comment:
Is this file needed? It's not used anywhere reached by the tests.

Reply:
Ah, yeah, this was an idea that I had that I gave up on.

Reply:
Removed